auth/server/src/main/java/com/seagates3/service/AccessKeyService.java

```diff
@@ -52,5 +53,41 @@ public static AccessKey createFedAccessKey(User user, long timeToExpire)
 
         return accessKey;
     }
+
+    /*
+     * Create access keys for the root user.
+     */
+    public static AccessKey createAccessKey(User user)
+            throws DataAccessException {
+        AccessKeyDAO accessKeyDAO = (AccessKeyDAO) DAODispatcher.getResourceDAO(
+                DAOResource.ACCESS_KEY);
+        AccessKey accessKey = new AccessKey();
+        accessKey.setUserId(user.getId());
+        accessKey.setId(KeyGenUtil.createUserAccessKeyId(true));
+        accessKey.setSecretKey(KeyGenUtil.generateSecretKey());
+        accessKey.setStatus(AccessKeyStatus.ACTIVE);
+
+        accessKeyDAO.save(accessKey);
```

Comment:
call AccessKey createAccessKey(User user, String accessKeyId,
            String secretKey) throws DataAccessException {

to save access key. This function can just generate access key

Reply:
Done